src/controls/HoverReactionsBar/components/reactionPicker/ReactionPicker.tsx

```diff
@@ -0,0 +1,199 @@
+            return [
+              ...emojis,
+              <RenderEmoji
+                key={index}
+                emoji={emojiInfo}
+                onSelect={(emoji) => {
+                  document.body.style.pointerEvents = ""
+                  onSelect(emoji, emojiInfo);
+                  onDismiss();
+                }}
+              />,
+            ];
+          });
+        }
+      }
+    },
+    [emojiList, onSelect, onDismiss, groups, returnType]
+  );
+
+  React.useEffect(() => {
+
+    const countainerBounds = target?.getBoundingClientRect() as DOMRect;
+    let pickerTopPosition = 0;
+    let pickerLeftPosition = 0;
+    // test if list of mentiosn can be displayed in full
+    if (countainerBounds?.top + PICKER_HEIGHT > window.innerHeight) {
+      pickerTopPosition = countainerBounds.top - PICKER_HEIGHT - 10;
+      if (pickerTopPosition < 0) {
+        pickerTopPosition = 0;
+      }
+    } else {
+      pickerTopPosition = countainerBounds?.bottom  + 10;
+      if (pickerTopPosition < 0) {
+        pickerTopPosition = 0;
+      }
+    }
+
+    if (countainerBounds?.left + PICKER_WIDTH > window.innerWidth) {
+      pickerLeftPosition = countainerBounds?.left - PICKER_WIDTH;
+    } else {
+      pickerLeftPosition = countainerBounds?.left;
+    }
+
+    setStyles({
+      top: `${pickerTopPosition}px`,
+      left: `${pickerLeftPosition}px`,
+      zIndex: "1000000",
+      display: isOpen ? "block" : "none",
+    });
+  }, [target, isOpen, pickerRef]);
+
+  React.useEffect(() => {
+
+    const listByGroup = getFluentEmojisByGroup();
+    groupsRef.current = listByGroup;
+    setGroups(listByGroup);
+  }, []);
+
+  React.useEffect(() => {
+    setRenderEmoji([]);
+    setSearchValue("");
+    loadEmoji(DEFAULT_GROUP);
+  }, [isOpen]);
+
+  const searchEmoji = React.useCallback(
+    (searchText: string) => {
+      isSearchingRef.current = true;
+      const mewlist = getFluentEmojis().filter((emojiInfo:IEmojiInfo) => {
+        if (emojiInfo) {
+          return emojiInfo.keywords?.some((keyword) => keyword.includes(searchText));
+        }
+        return false;
+      });
+      setRenderEmoji([]);
+      for (let index = 0; index < mewlist.length; index++) {
+        const emojiInfo = mewlist[index];
+        if (emojiInfo) {
+          setRenderEmoji((emojis) => {
+            return [
+              ...emojis,
+              <RenderEmoji
+                key={index}
+                emoji={emojiInfo}
+                onSelect={(emoji) => {
+                  onSelect(emoji, emojiInfo);
+                  onDismiss();
+                }}
+              />,
+            ];
+          });
+        }
+      }
+    },
+    [onSelect, onDismiss, groups, returnType]
+  );
+
+  return (
+    <>
+      <Card className={styles.card} style={pickerStyles} ref={pickerRef}>
+        <div className={styles.cardContent}>
+          <Input
+            value={searchValue}
+            placeholder="Search Emoji"
```

Comment:
Add placeholder string to loc files please

Reply:
fixed